.ruff.toml

```diff
@@ -0,0 +1,34 @@
+target-version = "py312"
+fix = true
```

Comment:
Does this fix the issues automatically on commit?

Reply:
Not on commit, but when you run `ruff check`.

Reply:
I won't in this PR to keep it more focussed (I'm also unfamiliar with the service), but perhaps something to consider.

A